wpeverest/everest-forms
# Tweak - Form locate option where form is activated

### All Submissions:


### Changes proposed in this Pull Request:

This PR is adding the form locate option in the title section of all form lists and, It gives the pages name list where that form located on the pages or posts.

### How to test the changes in this Pull Request:

1. Go to All forms.
2. Hover on any form title section -> click on locate form option
3. Check whether the page name is coming or not. Also, verify the page on click.

### Types of changes:


### Other information:



### Changelog entry

> Tweak - Form locate option where form is activated.
